apps/native/app/src/lib/passkeys/useAuthenticatePasskey.ts

```diff
@@ -0,0 +1,73 @@
+import { Passkey, PasskeyAuthenticationResult } from 'react-native-passkey'
+import { btoa } from 'react-native-quick-base64'
+import {
+  convertAuthenticationResultsToBase64Url,
+  convertBase64UrlToBase64String,
+  padChallenge,
+} from './helpers'
+import { useGetPasskeyAuthenticationOptionsLazyQuery } from '../../graphql/types/schema'
+import { preferencesStore } from '../../stores/preferences-store'
+
+const ONE_HOUR = 3600000
```

Comment:
~I like to include the math for variables like this to make it easy to verify, eg 60 * 60 * 1000. The value gets precompiled at build time.